internal/kinds/capp-revision/controllers/revision_controller.go

```diff
@@ -0,0 +1,105 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	cappv1alpha1 "github.com/dana-team/container-app-operator/api/v1alpha1"
+	"github.com/dana-team/container-app-operator/internal/kinds/capp-revision/action_mangers"
+	"github.com/dana-team/container-app-operator/internal/kinds/capp-revision/adapters"
+	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+	"reflect"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"time"
+)
+
+const RequeueTime = 5 * time.Second
+
+// CappRevisionReconciler reconciles a Capp object
+type CappRevisionReconciler struct {
+	Log logr.Logger
+	client.Client
+	Scheme        *runtime.Scheme
+	EventRecorder record.EventRecorder
+}
+
+func (r *CappRevisionReconciler) Reconcile(ctx context.Context, req reconcile.Request) (reconcile.Result, error) {
+	logger := log.FromContext(ctx).WithValues("CappName", req.Name, "CappNamespace", req.Namespace)
+	logger.Info("Starting Reconcile")
+	capp := cappv1alpha1.Capp{}
+	if err := r.Client.Get(ctx, req.NamespacedName, &capp); err != nil {
+		if errors.IsNotFound(err) {
+			logger.Info(fmt.Sprintf("Didn't find Capp: %s, from the namespace: %s", capp.Name, capp.Namespace))
```

Comment:
The Capp `name` and `Namespace` would already be logged since you used `WithValues` above, so you can remove it here.